src/ui-kit/form-controls/textarea/textarea.component.ts

```diff
@@ -147,7 +147,12 @@ export class SamTextareaComponent implements ControlValueAccessor {
 
     this.control.setValidators(validators);
     if (!this.useFormService) {
-      this.control.statusChanges.subscribe(() => {
+      this.control.valueChanges
+      .pipe(pairwise())
+      .subscribe(([prev, next]: [any, any]) => {
+          if (prev === '') {
```

Comment:
if we have a required validation, we're now marking the control as pristine. This no longer displays error messages for this specific validation - https://github.com/GSA/sam-ui-elements/blob/master/src/ui-kit/wrappers/label-wrapper/label-wrapper.component.ts#L179

Reply:
@diego-ruiz-rei Yes Diego, I am making this change to fix the issue for the IE browser. In IE the validation message is displaying by default.